app/controllers/facilities_controller.rb

```diff
@@ -38,7 +32,9 @@ def dashboard
   # GET /facilities
   def index
     @facilities = Facility.active.sorted
+    @recently_used_facilities = MostRecentlyUsedSearcher.new(acting_user).recently_used_facilities.sorted
```

Comment:
Should we create an aliased scope `alphabetized` for facilities? I do like that better than `sorted`. Probably want to leave `sorted` around for now because other stuff probably uses it. I'd be fine if we did that separately, though.